python/vyos/configdict.py

```diff
@@ -415,6 +415,11 @@ def get_interface_dict(config, base, ifname=''):
             bridge = is_member(config, f'{ifname}.{vif_s}.{vif_c}', 'bridge')
             if bridge: dict['vif_s'][vif_s]['vif_c'][vif_c].update(
                 {'is_bridge_member' : bridge})
+    
+    # Look for removed port mirroring settings
+    tmp = node_changed(config, ['mirror'])
```

Comment:
I still do not understand why is is required. If the `mirror` key is absent in the dict it is the same as remove mirror. Please change to logic to it was in the good old Perl days. Always assume port mirroring is gone.